static/app/utils/oxfordizeArray.tsx

```diff
@@ -11,30 +11,6 @@ const oxfordizeArray = (strings: string[]) =>
     ? strings.join(' and ')
     : [strings.slice(0, -1).join(', '), strings.slice(-1)[0]].join(', and ');
 
-export const oxfordizeElements = (elements: JSX.Element[]): JSX.Element => {
```

Comment:
What is the alternative to `oxfordizeElements`?
It is still used in [gsBanner](https://github.com/getsentry/getsentry/blob/master/static/getsentry/gsApp/components/gsBanner.tsx#L42) and [planSelect](https://github.com/getsentry/getsentry/blob/master/static/getsentry/gsApp/views/amCheckout/steps/planSelect.tsx#L23).

Reply:
Never mind, I just found https://github.com/getsentry/getsentry/pull/11715